ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/Ubuntu.yml

```diff
@@ -25,17 +28,28 @@ Build_Tool_Packages:
   - libffi-dev
   - libfreetype6-dev
   - libfontconfig1-dev
+  - libgl1-mesa-dev
+  - libgstreamer0.10-dev
+  - libgstreamer-plugins-base0.10-dev
   - libgmp3-dev
+  - libgtk2.0-dev
+  - libgtk-3-dev
+  - libjpeg-dev
   - libmpfr4
   - libmpfr4-dbg
   - libmpfr-dev
   - libmpfr-doc
+  - libpng-dev
+  - libudev-dev
   - libwww-perl
   - libx11-dev
   - libxext-dev
+  - libxml2-dev
   - libxrender-dev
+  - libxslt1-dev
   - libxt-dev
   - libxtst-dev
+  - libxxf86vm-dev
   - make
```

Comment:
Whats the purpose of the additional packages?

Reply:
FIrst thing I looked at this morning....answer is obvious `build OpenJFX on Linux` as the pr clearly states.

Reply:
Can we separate these out into a separate OpenJFX section so we know which packages were added for that please?

Reply:
Also we've moved our build machines to be CentOS/RHEL - is there a specific need for Ubuntu to build OpenJFX?

Reply:
(Addition to previous comment - according to https://wiki.openjdk.java.net/display/OpenJFX/Building+OpenJFX `yum install mercurial bison flex gperf ksh pkgconfig libpng12-devel libjpeg-devel libxml2-devel libxslt-devel systemd-devel glib2-devel  gtk2-devel libXtst-devel pango-devel freetype-devel` will do the trick)